WorkingWithText/Program.cs

```diff
@@ -14,24 +15,104 @@ public static class WorkingWithText
         // Do not use .Sort, it will cause the test to pass when it actually does not.
         public static bool IsConsecutive(string hyphenNum)
         {
-            return default;
+            bool valid = false;
+
+            try
```

Comment:
Was surprised to see a try-catch! However, a try-parse @ line 24 will work just the same.